HW05.Rmd

```diff
@@ -0,0 +1,153 @@
+
+```{r solve vorinostat}
+#cpd_name will have to be "vorinostat"
+vorinostat_search <- full_combo %>% #assign to dataframe vorinostat_search
+  filter(cpd_name == "vorinostat") %>% #only look at vorinostat compound rows
+  arrange(area_under_curve) #order so that smallest AUC at top
+
+vorinostat_search %>%
+  ggplot() + #plots plots plots plots plots plots EVERYBODY
+  aes(area_under_curve, cancer_type) + #look at AUC by cancer_type
+  geom_boxplot(color = "dodgerblue1", fill = "dodgerblue3") + #boxplots to demo mean vs outlier distinction
+  labs(x = "AUC", #area_under_curve
+       y = "Cancer Type", #cancer_type
+       title = "Vorinostat AUC by Cancer Type",
+       subtitle = "On average, it appears haematopoietic\nand lymphoid tissue have the lowest AUC") +
+  theme(plot.title = element_text(hjust = 0.5)) + #center title
+  theme(plot.subtitle = element_text(hjust = 0.5)) #center subtitle
+```
+
+Based on this plot, it seems on average haematopoietic and lymphoid tissue have the lowest AUC
+However, upper aerodigestive tract has that one super low AUC outlier, of note
+
+
+
+
+**Which compound is the prostate cancer cell line 22RV1 most sensitive to? (For 22RV1, which compound has the lowest AUC value?)**
+
+```{r solve 22RV1}
+line22RV1_search <- full_combo %>% #assign to dataframe line22RV1_search
+  filter(ccl_name == "22RV1") %>% #only look at ccl_name of 22RV1
+  arrange(area_under_curve) #order so that smallest AUC at top
+
+line22RV1_search %>% #use this dataframe to plot
+  slice_head(n = 10) %>% #I don't care about the ones with large AUCs. Top 10 smallest pls
+  ggplot() + #plots plots plots plots plots plots EVERYBODY
+  aes(area_under_curve, cpd_name) + #look at AUC by cpd_name
+  geom_point(color = "orange") + #point graph
+  labs(x = "AUC", #area_under_curve
+     y = "Compound Name", #cpd_name
+     title = "Compounds 22RV1 are Sensitive to",
+     subtitle = "It appears leptomycin B has the lowest AUC for 22RV1") +
+  theme(plot.title = element_text(hjust = 0.5)) + #center title
+  theme(plot.subtitle = element_text(hjust = 0.5)) #center subtitle
+```
+
+Based on this plot, it appears leptomycin B has the lowest AUC for 22RV1
+
+
+
+
+**For the 10 compounds that target EGFR, which of them has (on average) the lowest AUC values in the breast cancer cell lines?**
+
+```{r solve EGFR}
+EFGR_search <- full_combo %>% #assign to dataframe EFGR_search
+  filter(grepl("EGFR",gene_symbol_of_protein_target)) #only look at lines with EGFR
+
+EFGR_search %>%
+  ggplot() + #plots plots plots plots plots plots EVERYBODY
+  aes(area_under_curve, cpd_name) + #look at AUC by dpc_name
+  geom_boxplot(color = "green4", fill = "green1") + #boxplots to demo mean vs outlier distinction
+  labs(x = "AUC", #area_under_curve
+       y = "Compound Name", #cpd_name
+       title = "AUC of Compounds that Target EFGR",
+       subtitle = "On average, it appears afatinib has the lowest AUC") +
```

Comment:
I almost missed your answer to this question. Maybe you should write it outside the graphs like the other two questions